forge-gui/res/languages/es-ES.properties

```diff
@@ -3300,3 +3300,4 @@ lblWurm=Wurm
 lblYeti=Yeti
 lblZombie=Zombi
 lblZubera=Zubera
+lblWouldYouLikeToPlaneswalk=Would you like to Planeswalk ?
```

Comment:
I'd do `Would you like to planeswalk?` (no cap or extra space)

Reply:
Oops in English anyway... sometimes I get en-US and es-ES mixed. Other languages probably matter less  to fix in that way, as they're just placeholders.

Reply:
Yeah I also feel like leaving this as the last entry since the people who do localizations can better reorganise these entries